chain/solana/solana_test.go

```diff
@@ -57,7 +57,7 @@ var _ = Describe("Solana", func() {
 
 			// Mint some tokens.
 			time.Sleep(10 * time.Second)
-			mintAmount := uint64(10000000000) // 10 tokens.
+			mintAmount := uint64(1000000000) // 10 tokens.
```

Comment:
Do we need to change the comments as well?

Reply:
No, since we've changed the decimals from 9 to 8, no need.